doc/fork_support.md

```diff
@@ -0,0 +1,49 @@
+# Background #
+
+In Python multithreading is ineffective at concurrency for CPU bound tasks
+due to the GIL.
+(global interpreter lock).  Users use multiprocessing, subprocess, 
+and concurrent.futures.ProcessPoolExecutor, to work around the GIL.
+These modules call fork() underneath the hood. Various issues have
+been reported when using these modules with gRPC Python.
+Historically, we didn't support forking in gRPC, but some users seem
+to be doing fine until their code started to break on version 1.6.  This was
+likely caused by the addition of background c-threads and a background
+Python thread.
+
+# Current Status #
+## 1.7 ##
+A pthread_atfork() handler was added in 1.7 to automatically shut down
+the background c-threads when fork was called.  This does not shut down the
+background Python thread, so users could not have any open channels when
+forking().
+
+## 1.9 ##
+A regression was noted in cases where users are doing fork/exec. This
+was due to pthread_atfork() handler that was added in 1.7 to partially
+support forking in gRPC. A deadlock can happen when pthread_atfork
+handler is running, and an application thread is calling into gRPC.
+We have provided a workaround for this issue by allowing users to turn 
+off the handler using env flag ```GRPC_ENABLE_FORK_SUPPORT=False```.
+This should be set whenever a user expects to always call exec
+immediately following fork.  It will disable the fork handlers.
+
+## 1.11 ##
+The background Python thread was removed entirely.  This allows forking
+after creating a channel.  However, the channel cannot be used by both the
+parent and child process after the fork.  Additionaly, the process should
+not fork if there are any active RPCs on the channel.
+
+# Future Work #
+## 1.13 ##
+The workaround when using fork/exec by setting
+```GRPC_ENABLE_FORK_SUPPORT=False``` should no longer be needed.  Now the fork
+handlers are automatically not run when multiple threads are calling
+into gRPC.
+
+
+## 1.1x ##
+We would like to support forking and using the channel from both the parent
```

Comment:
I don't think we plan to support using the same channel in both parent and child, at least not now - rather, we want to support forking with active channels and active RPCs in the parent, with the child able to establish new (independent) gRPC connections. I'm going to file a new issue to track our plans for fork support, so it may be appropriate to drop this `1.1x` section entirely from this doc and just replace it with a link to the tracking issue.

Reply:
Removed this section.